packages/forms-web-app/src/dynamic-forms/dynamic-components/list-add-more/question.js

```diff
@@ -103,7 +105,10 @@ class ListAddMoreQuestion extends Question {
 	 * @returns {QuestionViewModel}
 	 */
 	prepQuestionForRendering(section, journey, customViewData) {
-		const answers = journey.response.answers[this.fieldName];
+		const isAddressQuestion = this.subQuestion instanceof AddressAddMoreQuestion;
+		const answers = isAddressQuestion
+			? journey.response.answers.SubmissionNeighbourAddress
```

Comment:
so hadn't realised this is the way we had gone for this, seems strange to have Neighbour in the DB name feel as if it should just be submission address, as it can be linked to any multi address question

Think we need to filter by question type though
So should the DB have question name as a column as well, and this be a function to filter by address?

Reply:
I have amended the table name to SubmissionAddress and updated relevant API calls etc.